examples/basics/Export_V1_to_V2_migration_support.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Can you remove the batch ID here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>